src/libraries/Microsoft.PowerFx.Connectors/Obsolete/ODataParameters.cs

```diff
@@ -42,8 +46,9 @@ public class ODataParameters
         // $search
         public string Search { get; init; } = null;
 
-        // $select
-        public string Select { get; init; } = null;
+        // $select        
+        // Select is encoded as a comma separate list of fields.
+        public IReadOnlyCollection<string> Select { get; init; } = null;
```

Comment:
We need to know the separate values so that we can encode as `c1,c2` and not encode the comma.